layouts/shortcodes/include.html

```diff
@@ -0,0 +1,14 @@
+{{$file := .Get "file"}}
+{{ $options := "linenos=false" }}
+{{- if (.Get "options") -}}
```

Comment:
Could we use the default function for the option stuff? See https://github.com/xoxys/hugo-geekdoc/blob/1df1bfe8f491e9bc93ecd5e2d21ed05aaae0788e/layouts/shortcodes/expand.html#L4

Reply:
good idea